### ISSUE TITLE
em to pt conversion for color contrast check

### ISSUE THREAD

**Issue by nekennedy** [daisy/ace#111]
Transformation of font size from ems to points when calculating contrast ratio _could_ be (but does not have to be) made more "ebookish". The current implementation uses the web standard of 1em=16px=12pt.

The problem is determining the correct transformation size so that if text is set so small that it will never hit 18pt, so does need the 4.5:1 contrast ratio. Lighthouse seems to use a different conversion formula, since a file that passes that test fails the ace check. Example file attached, but the relevant attributes are below:
background color: #f15e2b
text color: #ffffff
font size: .9em

The following success technique would suggest that the ability to easily resize the font within a reading system is sufficient to meet SC 1.4.3 with a 3:1 contrast ratio at 1em. ace-core is currently requiring a ratio of 4.5:1 for text this size.
[G174: Providing a control with a sufficient contrast ratio that allows users to switch to a presentation that uses sufficient contrast](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G174)

[color_lightouse_pass.epub.zip](https://github.com/daisy/ace/files/1454874/color_lightouse_pass.epub.zip)

**Comment by mattgarrish** [comment]
A user agent that supports resizing of text is only an advisory technique:
https://www.w3.org/WAI/WCAG20/quickref/#qr-visual-audio-contrast-contrast

That means, on its own, it is not sufficient to count as a pass. In the referenced technique, the author has to provide a link to an alternative page that has text sized to specified ratio.

Users who need large print are not necessarily users who need high contrast, so there can't be an expectation that the user has to read a large print file in order to consume the content. If someone needs large print, however, then the contrast doesn't have to be as high.
